cocos/game/game.ts

```diff
@@ -518,6 +519,27 @@ export class Game extends EventTarget {
         director.tick(this.frameTime / 1000);
     }
 
+    /**
+     * @en Called by the engine to pause the game and will not be automatically resumed.
+     * @zh 提供给引擎调用暂停游戏接口，不会主动恢复。
```

Comment:
The comment seems not correct. If paused by engine, then will be resumed by engine automatically.